Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ImageEx/ImageExPage.xaml.cs

```diff
@@ -67,6 +69,23 @@ private async void Load()
                 AddImage(false, false, true);
             });
 
+            SampleController.Current.RegisterNewCommand("Lazy loading sample (17763 or higher supported)", (sender, args) =>
```

Comment:
Instead, only register this command if `IsLazyLoading` is supported, then you don't need the quantify text (call that out in the docs).